region/maridia/outer.json

```diff
@@ -1994,6 +2102,84 @@
                   ],
                   "note": "The second jump is doable by combining a stationary mid-air mockball with a mid-air springball jump.",
                   "devNote": "The second jump is actually doable with just a regular SpringBall jump but is ridiculously tight."
+                },
+                {
+                  "name": "Suitless Water Shinecharge Vertical Spark",
+                  "notable": false,
+                  "requires": [
+                    "canWaterShineCharge",
+                    {"or": [
+                      {"canShineCharge": {
+                        "usedTiles": 33,
+                        "shinesparkFrames": 43,
+                        "excessShinesparkFrames": 9,
+                        "openEnd": 2
+                      }},
+                      {"and": [
+                        "canMidairShinespark",
+                        {"canShineCharge": {
+                          "usedTiles": 33,
+                          "shinesparkFrames": 38,
+                          "excessShinesparkFrames": 9,
+                          "openEnd": 2
+                        }}
+                      ]}
+                    ]},
+                    {"adjacentRunway": {
+                      "fromNode": 3,
+                      "usedTiles": 0.5,
+                      "physics": ["normal"],
+                      "overrideRunwayRequirements": true,
+                      "useFrames": 80
+                    }}
+                  ],
+                  "note": [
+                    "Start the dash as close to the door transition as possible in order to have a larger window for storing the shinecharge.",
+                    "Spark up and fall onto the platform left of the fish."
+                  ]
+                },
+                {
+                  "name": "Suitless Water Shinecharge Diagonal Spark",
+                  "notable": false,
+                  "requires": [
+                    "canWaterShineCharge",
+                    {"or": [
+                      {"canShineCharge": {
+                        "usedTiles": 33,
+                        "shinesparkFrames": 43,
+                        "excessShinesparkFrames": 26,
+                        "openEnd": 2
+                      }},
+                      {"and": [
+                        "canMidairShinespark",
+                        {"canShineCharge": {
+                          "usedTiles": 33,
+                          "shinesparkFrames": 38,
+                          "excessShinesparkFrames": 25,
+                          "openEnd": 2
+                        }}
+                      ]}
+                    ]},
+                    {"adjacentRunway": {
+                      "fromNode": 3,
+                      "usedTiles": 0.5,
+                      "physics": ["normal"],
+                      "overrideRunwayRequirements": true,
+                      "useFrames": 80
+                    }},
+                    {"or": [
```

Comment:
A movement tech is needed here.  A no movement tech path would go to 5 and come back.
h_crouchjumpdowngrab
ice+plasma
springballjump
Hijump would use a different strat.